README.md

```diff
@@ -57,24 +73,29 @@ worker.get('/myResource.html', function(request, response) {
 });
 ```
 
-## Why I would like to support more than just handling requests?
+TODO: a working code example that can be executed to see how 'Your code' would actually look like
```

Comment:
We could use the following:
`importScripts('./sww.js');
var worker = new self.ServiceWorkerWare();
worker.get('/hello.html', function(request, response) {
  return Promise.resolve(new Response('Hello world!), { headers: {'Content-Type': 'text/html'} });
}
worker.init();`